bucket/lockhunter.json

```diff
@@ -2,19 +2,83 @@
     "version": "3.4.3",
     "description": "Delete files blocked by something you do not know.",
     "homepage": "https://lockhunter.com/index.htm",
-    "license": "Unknown",
+    "license": "Freeware",
+    "depends": "sudo",
     "url": "https://lockhunter.com/assets/exe/lockhuntersetup_3-4-3.exe",
     "hash": "2c0d52dca3e5ce9cfc8062cb72c7235e3c6ff650242fcb7d46a892d602dd3dd1",
     "innosetup": true,
-    "architecture": {
-        "64bit": {
-            "pre_install": "Get-ChildItem \"$dir\" '*,1.*' | Rename-Item -Path { $_.Fullname } -NewName { $_.Basename.TrimEnd(',1') + $_.Extension }"
-        },
-        "32bit": {
-            "pre_install": "Get-ChildItem \"$dir\" '*,2.*' | Rename-Item -Path { $_.Fullname } -NewName { $_.Basename.TrimEnd(',2') + $_.Extension }"
-        }
+    "notes": "Add Lockhunter as context menu option by running: reg import \"$dir\\install-context.reg\"",
+    "pre_install": [
+        "If ($architecture -eq '64bit') {",
+        "   Get-ChildItem \"$dir\" '*,1.*' | Rename-Item -Path { $_.Fullname } -NewName { $_.Basename.TrimEnd(',1') + $_.Extension }",
+        "} Else {",
+        "   Get-ChildItem \"$dir\" '*,2.*' | Rename-Item -Path { $_.Fullname } -NewName { $_.Basename.TrimEnd(',2') + $_.Extension }",
+        "}",
+        "$url = \"https://raw.githubusercontent.com/ScoopInstaller/Extras/master/scripts/lockhunter/install-context.reg\"",
```

Comment:
Why not add them as https://github.com/ScoopInstaller/Extras/tree/master/scripts/lockhunter/install-context.reg and
https://github.com/ScoopInstaller/Extras/tree/master/scripts/lockhunter/uninstall-context.reg

and them change
`$dir\\install-context.reg`
to
`$scoopdir/apps/scoop/current/scripts/lockhunter/install-context.reg`
per https://github.com/ScoopInstaller/Scoop/wiki/Pre--and-Post-install-scripts#variables

That makes more sense them unnecessarily downloading files that are already in the repo.

Reply:
I think you mean from `$scoopdir\scoop\buckets\extras\scripts\lockhunter`?

Yeah didn't think of that, so just copy those 2 files from the bucket scripts directory to `$dir`.

After that should I be checking the sha256 for these?

Reply:
done